### PR TITLE
Update refpoint to include new bug fixes

### PR DESCRIPTION
**Pull request by ravi861**
Signed-off-by: Ravi Vantipalli <ravi.vantipalli@intel.com>

### CODE REVIEW

File: docs/ipdk-tofino.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
 
 ### Intel P4Studio
 
-Get Intel P4Studio SDE version 9.11. The untarred directory is henceforth referred to as **sde**. If using an Intel Tofino Reference platform, please also download the BSP package.
+Get Intel P4Studio SDE. The untarred directory is henceforth referred to as **sde**. If using an Intel Tofino Reference platform, please also download the BSP package.
```
Comment (nupuruttarwar) [inline docs/ipdk-tofino.md:41]: Say 9.11 or later since TDI was not available before 9.11?

Comment (ravi861) [reply docs/ipdk-tofino.md:41]: At this point 9.11 is not compatible anymore and we have not released 9.12. We have to go back and find a compatible refpoint and tag it for 9.11 compatibility.
Either way, SDE compatibility is to be dealt offline. This document does not need to hold that information.

Comment (nupuruttarwar) [reply docs/ipdk-tofino.md:41]: Then maybe we should change the default target of networking recipe to DPDK?

Comment (ravi861) [reply docs/ipdk-tofino.md:41]: I don't see how it matters but we can do that. In my opinion, there shouldn't be a default target. Let the user explicitly specify what they are interested in building.

Comment (nupuruttarwar) [reply docs/ipdk-tofino.md:41]: Can you make that change - https://github.com/ipdk-io/networking-recipe/blob/main/CMakeLists.txt#L51

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Pull request merged**